docs/tutorials/host_apt_news.rst

```diff
@@ -0,0 +1,225 @@
+Host your own APT News
+**********************
+
+In this tutorial, you will learn how to host your own APT news and how to
+configure your Ubuntu machines to use it.
+
+APT news is used to display custom messages within the ``apt upgrade`` command,
+and can be a good way of delivering messages that are related to APT updates
+to users.
+
+.. Why we use Multipass + command to install it
+.. include:: ./common/install-multipass.txt
+
+Tutorial Architecture
+=====================
+
+To demonstrate how APT news works, we will create three containers inside a
+Multipass VM. We will configure one of them to be the APT news server, and we
+will configure the other two to consume APT news from that server. It will
+look something like this:
+
+.. image:: host-apt-news.drawio.png
+   :align: center
+
+Command snippets throughout this tutorial will include a shell prompt to make
+it clear on which machine the command will be executed.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Prompt
+     - Machine
+   * - ``you@yourcomputer:~$``
+     - Your computer
+   * - ``ubuntu@apt-news-tutorial:~$``
+     - Multipass VM named ``apt-news-tutorial``
+   * - ``root@apt-news-server:~#``
+     - LXD container named ``apt-news-server``
+   * - ``root@client-noble:~#``
+     - LXD container named ``client-noble``
+   * - ``root@client-jammy:~#``
+     - LXD container named ``client-jammy``
+
+
+Create the local VM and containers
+==================================
+
+Begin by launching an Ubuntu 24.04 Multipass VM with the following command:
+
+.. code-block:: console
+
+   you@yourcomputer:~$ multipass launch noble --disk 10G --name apt-news-tutorial
+
+Access the VM:
+
+.. code-block:: console
+
+   you@yourcomputer:~$ multipass shell apt-news-tutorial
+   ubuntu@apt-news-tutorial:~$
```

Comment:
Split and explain prompt change

Reply:
just this first time

Reply:
e.g. when you see "Welcome to Ubuntu 24.04 LTS" at the top and the prompt change, you know you're in the VM